src/Properties.ts

```diff
@@ -0,0 +1,28 @@
+import { Logger } from '@hmcts/nodejs-logging'
+import * as fs from 'fs'
+
+const logger = Logger.getLogger('applicationRunner')
+
+export function addTo (config: any, mountPoint: fs.PathLike = '/mnt/secrets/', propertiesPrefix: string = 'keyVault') {
+  logger.info(`Reading properties from volume: '${mountPoint}'`)
+  config[propertiesPrefix] = fs.readdirSync(mountPoint, null)
+    .reduce((obj, dir) => addDir(dir, obj, mountPoint), {})
+}
+
+function addDir (dir: string, obj: any, mountPoint: fs.PathLike): any {
+  obj[dir] = readDirectories(mountPoint, dir).reduce((values, file) => addFile(values, file, mountPoint, dir), {})
+  return obj
+}
+
+function addFile (values: any, file: string, mountPoint: fs.PathLike, dir: string) {
+  values[file] = readFile(mountPoint, dir, file).trim()
+  return values
+}
+
+function readFile (mountPoint: fs.PathLike, dir: string, file: string) {
```

Comment:
always declare the return type even if its any, can you add the lint rule `noImplicitAny` please?

Reply:
sure ... thought it was inferred like scala and kotlin

Reply:
it's inferred but slows it down and makes it more difficult to review, often times its accidental so the recommendation is to always be explicit, you can use any, but you should declare you are intentionally doing it